networking/manager.go

```diff
@@ -1036,7 +1036,7 @@ func (this *networkManagerInstance) setupInterfaces() (err error) {
 			task.Src = "SetupExistingInterfaces"
 			task.Op = op
 
-			err = this.SubmitTask(task)
```

Comment:
Can you add a git-commit message as to why this revert was needed?  Or better yet, amend the commit that introduced this change if that commit is also contained within this PR?

Reply:
Done.